modules/administration-guide/pages/the-woopra-telemetry-plugin.adoc

```diff
@@ -1,24 +1,24 @@
 :_content-type: PROCEDURE
-:description: The Woopra telemetry plug-in
+:description: The Woopra telemetry plugin
 :keywords: extensions, telemetry
-:navtitle: The Woopra telemetry plug-in
-:page-aliases: extensions:the-woopra-telemetry-plug-in.adoc
+:navtitle: The Woopra telemetry plugin
+:page-aliases: extensions:the-woopra-telemetry-plugin.adoc
```

Comment:
Are you sure you want to change this line?

Reply:
Mistake, indeed